pkg/ccl/streamingccl/streamingest/stream_ingestion_job_test.go

```diff
@@ -562,6 +566,11 @@ func TestCutoverCheckpointing(t *testing.T) {
 
 	jobutils.WaitForJobToRun(c.T, c.SrcSysSQL, jobspb.JobID(producerJobIDInt))
 	jobutils.WaitForJobToRun(c.T, c.DestSysSQL, replicationJobID)
+
+	// Assert that our counters have been incremented.
+	counts := telemetry.GetFeatureCounts(telemetry.Raw, telemetry.ResetCounts)
+	require.GreaterOrEqual(t, counts["physical_replication.started"], int32(1))
```

Comment:
why isn't this `require.Equal()`?

Reply:
"something something txn retries"

Reply:
wait, but even if we increment when `err == nil`?? Thats odd. I wish we could test that this thing only gets incremented 1 time, most of the time.